presto-product-tests/conf/docker/singlenode-sqlserver/compose.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+source ${BASH_SOURCE%/*}/../common/compose-commons.sh
+
+docker-compose \
+-f ${BASH_SOURCE%/*}/../common/standard.yml \
+-f ${BASH_SOURCE%/*}/../common/jdbc_db.yml \
+-f ${BASH_SOURCE%/*}/../common/cassandra.yml \
```

Comment:
this should go in after https://github.com/prestodb/presto/pull/7684, so that we can get rid of `cassandra.yml` here